### PR TITLE
Change raw-media-types to accept a string of comma separated MIME types

### PR DESCRIPTION
**Pull request by Dansvidania**
should solve #1369

### CODE REVIEW

File: src/PostgREST/Config.hs [hunk 1]
```diff
@@ -180,11 +180,11 @@ readOptions = do
...
 
-    optionalListOfText :: C.Key -> C.Parser C.Config [Text]
-    optionalListOfText k =
-      C.optional k (C.list C.string) >>= \case
+    optionalCommaSeparatedList :: C.Key -> C.Parser C.Config [Text]
+    optionalCommaSeparatedList k =
+      C.optional k C.string >>= \case
         Nothing -> pure []
-        Just types -> pure types
+        Just commaSeparatedValues -> pure $ strip <$> splitOn "," commaSeparatedValues
```
Comment (steve-chavez) [inline src/PostgREST/Config.hs]: Down below there's a `splitExtraSearchPath` function which has the same logic. Perhaps that function can be renamed and reused here.

File: src/PostgREST/Config.hs [hunk 2]
```diff
@@ -283,7 +283,7 @@ readOptions = do
           |# root-spec = "stored_proc_name"
           |
           |## content types to produce raw output
-          |# raw-media-types=["image/png","image/jpg"]
+          |# raw-media-types="image/png,image/jpg"
```
Comment (steve-chavez) [inline src/PostgREST/Config.hs]: nitpick: Add a space between the types.`"image/png, image/jpg"`

File: src/PostgREST/Config.hs (hunk 1 above)
Comment (Dansvidania) [reply src/PostgREST/Config.hs]: I was in doubt rregarding whether refactoring that code would be out of the scope of this PR, will fix. Thanks for the feedback

File: src/PostgREST/Config.hs (hunk 2 above)
Comment (Dansvidania) [reply src/PostgREST/Config.hs]: Ok

**Comment by steve-chavez** [comment]
LGTM! Regarding refactoring, I think we should do it whenever we can.

Merging..

**Pull request merged**